setup.py

```diff
@@ -127,13 +127,11 @@
             'cernopendata.cli:cli',
         ],
         'invenio_assets.webpack': [
-            'cernopendata_theme = cernopendata.modules.theme.webpack:js',
+            'cernopendata_theme = cernopendata.modules.theme.webpack:theme',
         ],
         # 'invenio_assets.bundles': [
-            # 'cernopendata_js = cernopendata.modules.theme.bundles:js',
             # 'cernopendata_front_js = cernopendata.modules.theme.bundles'
             # ':front_js',
-            # 'cernopendata_theme_css = cernopendata.modules.theme.bundles:css',
```

Comment:
Minor: Maybe we can start deleting bundles from `bundles.py`? so we can keep track of what is migrated.